enacts/flex_fcst/maproom.py

```diff
@@ -439,21 +245,22 @@ def local_plots(marker_pos, start_date, lead_time):
         ).rename("obs_ppf")
         # Obs quantiles
         obs_quant = obs.quantile(quantiles, dim="T")
-
         # Forecast CDF
-        fcst_q, fcst_mu = xr.broadcast(quantiles, fcst_mu)
+        fcst_q, fcst_mu = xr.broadcast(quantiles, fcst_ds["deterministic"])
```

Comment:
until then, could work with the whole fcst_ds. From then, need to get the individual variables of interest